selfdrive/car/honda/carstate.py

```diff
@@ -235,9 +234,9 @@ def update(self, cp, cp_cam, cp_body):
         acc_hud = cp_cam.vl["ACC_HUD"] if self.CP.carFingerprint in HONDA_BOSCH_RADARLESS else cp.vl["ACC_HUD"]
         ret.cruiseState.nonAdaptive = acc_hud["CRUISE_CONTROL_LABEL"] != 0
         ret.cruiseState.standstill = acc_hud["CRUISE_SPEED"] == 252.
-
         # on certain cars, CRUISE_SPEED changes to imperial with car's unit setting
-        conversion_factor = CV.MPH_TO_MS if self.CP.carFingerprint in HONDA_BOSCH_RADARLESS and not self.is_metric else CV.KPH_TO_MS
+        ## TODO: verify with certainty. initial test route checks show that Bosch sets this unit according to the vehicles origin
```

Comment:
@sshane If you'd like this to left out for now or be reworked to be just a change for the Odyssey, I'd support that. Of the test routes for Bosch models, the only two that didn't have the Imperial Unit set were the Honda E (UK) and the Civic Diesel (Netherlands). I'll be working on a script to actually look at the set speed values too but it's not done yet.

Reply:
yeah can make the minimum amount of changes for this car port and open new ones to refactor things?